bigtable/cmd/cbt/testing.go

```diff
@@ -48,3 +54,28 @@ func captureStdout(f func()) string {
 	w.Close()
 	return <-outC
 }
+
+func assertEqual(t *testing.T, got, want interface{}, opts ...cmp.Option) {
+	if !testutil.Equal(got, want, opts...) {
+		_, fpath, lno, ok := runtime.Caller(1)
+		if ok {
+			_, fname := filepath.Split(fpath)
+			t.Errorf("%s:%d: Didn't match:\nGot: %s\nExpected: %s",
```

Comment:
nit: usually we use the words got/want. I am not sure if the extra newline formatting is needed here. See above link or reference other test files in the package for some examples.

Reply:
@codyoss I'm not sure what's being requested.  I'd be happy to change this to specifically use the got/want language, however, in the previous comment above, you seemed to be suggesting getting rid of the assert function altogether.  Honestly, I'd rather keep it, as I think it avoids a lot of boilerplate.

WRT newlines, I found them helpful for my use cases, as the values being compared were often multi-line text.